k8s/jobs/elasticSearchInitJob.yaml

```
@@ -0,0 +1,20 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: elasticsearch-init-job-
+  namespace: adhoc-jobs
+spec:
+  template:
+    metadata:
+      name: elasticsearch-init-job
+    spec:
+      containers:
+        - name: elasticsearch-init-job
+          command:
+            - 'bash'
+            - '-c'
+            - |
+                MW_INSTALL_PATH=/var/www/html/w/ php \
+                  /var/www/html/w/extensions/CirrusSearch/maintenance/UpdateSearchIndexConfig.php \
+                  --cluster="${CLUSTER_NAME:-all}"
+          restartPolicy: Never
```

I failed to apply this thing locally. Error was `Error from server (BadRequest): error when creating "STDIN": Job in version "v1" cannot be handled as a Job: strict decoding error: unknown field "spec.template.spec.containers[0].restartPolicy"`

I think restartPolicy should be on the whole pod not the container. However I wonder why we want to change the restart policy? If I understand correctly running this job twice is fine and we'd probably rather in the event of temporarily slow/timeout DB etc. this thing retries than we have to manually go and kick it off again?

```suggestion
      restartPolicy: Never
```

---

So we'd make it `OnFailure` instead?